corc-cascading/src/test/java/com/hotels/corc/cascading/SearchArgumentFactoryTest.java

```diff
@@ -44,20 +48,34 @@
 
   private SearchArgumentFactory.Builder builder;
 
+  //Don't know how to get the type right for this setup
   @Before
   public void setup() {
     builder = new Builder(mockInternal);
     when(mockInternal.startNot()).thenReturn(mockInternal);
-    when(mockInternal.lessThan(anyString(), any())).thenReturn(mockInternal);
-    when(mockInternal.lessThanEquals(anyString(), any())).thenReturn(mockInternal);
+    when(mockInternal.lessThan(anyString(), same(toType(ONE)), any())).thenReturn(mockInternal);
+    when(mockInternal.lessThanEquals(anyString(), same(toType(ONE)), any())).thenReturn(mockInternal);
     when(mockInternal.end()).thenReturn(mockInternal);
   }
 
+    //Possibly moving this
+    static PredicateLeaf.Type toType(Fields fields) {
```

Comment:
Hopefully we can remove this entirely, if not we can instead use the Utils class I mentioned above.